projects/storefrontstyles/scss/components/configurator/_index.scss

```diff
@@ -1 +1,3 @@
-@import 'attribute';
+@import 'config-attribute-radio-button';
+@import 'config-attribute-header';
+@import 'config-form';
```

Comment:
Since we have this file `components/configurator/commons/_index.scss` 
we are missing its precedence here as `@import './commons/index';`
instead of having it in this other file `components/index.scss` where all instances are only one level in.

![Screen Shot 2019-09-27 at 11 19 24 AM](https://user-images.githubusercontent.com/15113219/65781100-053fa580-e119-11e9-93ed-cf0df2cba9ff.png)

Reply:
Done